ocw-cli/cli_app.py

```diff
@@ -1232,11 +1180,11 @@ def settings_screen(header):
          if option == '4':
               screen.addstr(25, x/2, "Enter Spatial Gridding Option [Model, Observation or User]:")
               new_spatial_grid_option = screen.getstr()
-              if new_spatial_grid_option.lower() == 'model':
-                   spatial_grid_option = 'Model'
-                   spatial_grid_setting = model_lat_res
-                   note = "Spatial gridding option has changed successfully to {0}".format(spatial_grid_option)
-              elif new_spatial_grid_option.lower() == 'observation':
+              #if new_spatial_grid_option.lower() == 'model':
```

Comment:
Well, actually OCW cannot properly handle most of RCM's grid points which use curvilinear coordinates. So for now, it is safe to leave 'Observation' and 'User' options only. NARCCAP and CORDEX-AF datasets that we are using as examples are already regridded. They use totally different grid points from what users can download from ESGF.